CONTACTING-OPS.md

```diff
@@ -23,4 +23,17 @@ If you have exhausted the contact steps above, move on to these steps only for e
 1. Call or text specific Ops engineers. [Contact info](https://docs.google.com/document/d/1EDYhWYipUluzG6K8S-W4clsAGInm2RdjkpKq9Lw_dhE/edit).
    * If possible, pick an engineer who is in the middle of their work day over an engineer who is likely asleep. Timezone information is in [Contact info](https://docs.google.com/document/d/1EDYhWYipUluzG6K8S-W4clsAGInm2RdjkpKq9Lw_dhE/edit)
    * Repeat until you've reached an Ops engineer, or exhausted the list of Ops engineers (likely-awake or otherwise).
-1. Slack, Skype, text, or call Sandra, Colin, or Gregg. These people may know where to find an Ops engineer.
+1. [#ops in Slack](https://raisingthefloor.slack.com/messages/ops/), Skype, text, or call Sandra, Colin, or Gregg. These people may know where to find an Ops engineer.
+1. Email `ops at raisingthefloor dot org`. Please, attach as much detail of the issue as you can. A good template for such email could be:
+
+```
+Subject: [Type here a short description of the issue]
+Body of the message:
+
+* Location of the issue
```

Comment:
"URL" instead of "location"?

Reply:
With "location" I meant the place where the PC is in order to identify the installation, end points, etc.